mmcv/ops/nms.py

```diff
@@ -305,7 +305,23 @@ def nms_match(dets, iou_threshold):
         return [np.array(m, dtype=np.int) for m in matched]
 
 
-def nms_rotated(boxes, iou_thr, labels=None, multi_label=False):
+def nms_rotated(boxes, iou_threshold, labels=None, multi_label=False):
```

Comment:
The `score` argument is passed explicitly in `nms` and `soft_nms`. Can we support this style as well?